src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -120,13 +125,13 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
       isPublic: true,
       language: 'en',
       category: null,
-      licenseCode: null,
+      licenseCode: 1001,
```

Comment:
Do we need it here since it's already declared in provider? If yes, maybe we could move it to some common const? WDYT?
https://github.com/Joystream/atlas/pull/1347/files#diff-6ac1f2b87ffac0734ea21a1d15a11969a70658bce02cfde51fb3392e582e5819R73

Reply:
That's a good point, fixed. I'm a little worried that I could break something here though. I'm just surprised, why we have this doubled code here.
Could you also test this? Just play around - check if the draft is saved properly, check if you can upload video with correct data, etc.
I think everything should work, but it's possible that I could miss something.